scripts/api/user.ts

```diff
@@ -6,6 +6,12 @@ function hasPrivilege(privilege: string): boolean {
     return privileges.userHasPrivileges({[privilege]: 1});
 }
 
+function getCurrentUserId(): string {
```

Comment:
reuse this one https://github.com/superdesk/superdesk-client-core/blob/authoring-react-post-broadcasting/scripts/core/superdesk-api.d.ts#L2861

in the implementation it's inlined, so extract into a separate function

Reply:
How should I import this?

Reply:
https://github.com/superdesk/superdesk-client-core/blob/authoring-react-post-broadcasting/scripts/core/get-superdesk-api-implementation.tsx#L337

look at implementation of this. First you extract it into a function and then use that function anywhere in the code AND also expose via extensions API